compiler/main/driver.cpp

```diff
@@ -1421,6 +1339,7 @@ static ArgumentDescription arg_desc[] = {
  {"help-settings", ' ', NULL, "Current flag settings", "F", &fPrintSettingsHelp, "", driverSetHelpTrue},
  {"license", ' ', NULL, "Show license", "F", &fPrintLicense, NULL, NULL},
  {"print-chpl-home", ' ', NULL, "Print CHPL_HOME and exit", "F", &fPrintChplHome, NULL,NULL},
+ {"print-bootstrap-commands", ' ', NULL, "Print the installation prefix, if any, and exit", "F", &fPrintBootstrapCommands, NULL,NULL},
```

Comment:
If this flag does as it says, should it maybe be named `print-install-prefix` or similar? We might want to take a quick poll on the flag name.